.test-infra/jenkins/job_PostCommit_Java_Nexmark_Gearpump.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import CommonTestProperties.Runner
```

Comment:
Please add ```import CommonTestProperties.TriggeringContext``` - without it the job won't compile ("seed_job" fails).

Reply:
See: https://builds.apache.org/job/beam_SeedJob/3524/console for the logs. You can also rerun the seed job by typing "Run seed job" in a comment of this PR to see if everything compiles well. 

... and if you're discouraged by the rather slow feedback loop to see if the job compiles well, you can use a dockerized jenkins instance for testing it: https://github.com/apache/beam/tree/master/.test-infra/dockerized-jenkins